cocotb/__init__.py

```diff
@@ -96,31 +96,31 @@ def _reopen_stream_with_buffering(stream_name):
 # so that cocotb.scheduler gives you the singleton instance and not the
 # scheduler package
 
-scheduler = None  # type: cocotb.scheduler.Scheduler
+scheduler: Optional[cocotb.scheduler.Scheduler] = None
 """The global scheduler instance."""
 
-regression_manager = None  # type: cocotb.regression.RegressionManager
+regression_manager: Optional[cocotb.regression.RegressionManager] = None
 """The global regression manager instance."""
 
-argv = None  # type: List[str]
+argv: Optional[List[str]] = None
 """The argument list as seen by the simulator"""
 
-argc = None  # type: int
+argc: Optional[int] = None
 """The length of :data:`cocotb.argv`"""
 
-plusargs = None  # type: Dict[str, Union[bool, str]]
+plusargs: Optional[Dict[str, Union[bool, str]]] = None
 """A dictionary of "plusargs" handed to the simulation. See :make:var:`PLUSARGS` for details."""
 
-LANGUAGE = os.getenv("TOPLEVEL_LANG")  # type: str
+LANGUAGE: str = os.getenv("TOPLEVEL_LANG")
```

Comment:
```suggestion
LANGUAGE: Optional[str] = os.getenv("TOPLEVEL_LANG")
```

`getenv` can return `None`.

Reply:
Fixed.